deploy/quickstart.sh

```diff
@@ -0,0 +1,64 @@
+#!/bin/bash
+SAF_PROJECT=${SAF_PROJECT:-sa-telemetry}
+SAF_CONFIG=${SAF_CONFIG:-configs/default.bash}
+source ${SAF_CONFIG}
+
+oc new-project "${SAF_PROJECT}"
+oc create -f - <<EOF
+apiVersion: operators.coreos.com/v1
+kind: OperatorSource
+metadata:
+  name: redhat-service-assurance-operators
+  namespace: openshift-marketplace
+spec:
+  type: appregistry
+  endpoint: https://quay.io/cnr
+  registryNamespace: redhat-service-assurance
+  displayName: Service Assurance Operators
+  publisher: Red Hat (CloudOps)
+
+---
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: amq7-cert-manager
+  namespace: openshift-operators
+spec:
+  channel: alpha
+  installPlanApproval: Automatic
+  name: amq7-cert-manager
+  source: redhat-operators
+  sourceNamespace: openshift-marketplace
+  startingCSV: amq7-cert-manager.v1.0.0
+
+---
+
+apiVersion: operators.coreos.com/v1alpha2
+kind: OperatorGroup
+metadata:
+  name: ${SAF_PROJECT}-og
+  namespace: ${SAF_PROJECT}
+spec:
+  targetNamespaces:
+  -  ${SAF_PROJECT}
+
+---
+
+apiVersion: operators.coreos.com/v1alpha1
+kind: Subscription
+metadata:
+  name: serviceassurance-operator-alpha-redhat-service-assurance-operators-openshift-marketplace
+  namespace: ${SAF_PROJECT}
+spec:
+  channel: alpha
```

Comment:
beta

Reply:
All of these should be available on merge or rebase from master.

Reply:
I checked before, see https://github.com/redhat-service-assurance/telemetry-framework/blob/master/deploy/quickstart.sh#L54

Reply:
This is a great argument for merging the repos! :) Updating now.

Reply:
Now with the quickstart in this repo, I might be able to eliminate the duplication and have it re-use artifacts from the source like the old quickstart did.